### PR TITLE
chapter 6

### PR DESCRIPTION
**Pull request by jabirG**

### CODE REVIEW

File: chapter-6.qmd [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+##Solutions
+
+
+```{r}
+library(tidyverse)
+```
+
+## 6.2.1 Exercises
+
+1.Using prose, describe how the variables and observations are organised in each of the sample tables
```
Comment (jabirG) [inline chapter-6.qmd:19]:
```suggestion
1.Using prose, describe how the variables and observations are organised in each of the sample tables.
```

File: chapter-6.qmd [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+```{r}
+
+table4a |>
+  pivot_longer(
+    cols = -country,
+    names_to = "year",
+    values_to = "n"
+  )
+
```
Comment (jabirG) [inline chapter-6.qmd]:
```suggestion
```

File: chapter-6.qmd [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    table2 |>
+      pivot_wider(
+        names_from = type,
+        values_from = count
+      ) |> mutate(rate = cases/population * 10000)
+    ```
+
+```{r}
+
```
Comment (jabirG) [inline chapter-6.qmd]:
```suggestion
```

File: chapter-6.qmd [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Data tidying"
+---
+
+## Notes
+
+Using prose, describe how the variables and observations are organised in each of the sample tables.
+
+
+##Solutions
```
Comment (mine-cetinkaya-rundel) [inline chapter-6.qmd:10]: Remove these

File: chapter-6.qmd [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+
+```{r}
+library(tidyverse)
+```
+
+## 6.2.1 Exercises
+
+1.Using prose, describe how the variables and observations are organised in each of the sample tables.
```
Comment (mine-cetinkaya-rundel) [inline chapter-6.qmd:19]: This question is not answered?

File: chapter-6.qmd [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+2.
```
Comment (mine-cetinkaya-rundel) [inline chapter-6.qmd:21]: Some of these questions are not answered. Can you organize it so that each part is answered underneath its label?

**Review by mine-cetinkaya-rundel — CHANGES_REQUESTED** [review]

**Comment by mine-cetinkaya-rundel** [comment]
Thank you @jabirG -- answer updates at c1865975577018d26b394b457509a6faf7f81499 if you'd like to review!

**Pull request merged**